src/Server/middleware/linkHeadersMiddleware.ts

```diff
@@ -24,7 +21,6 @@ export function linkHeadersMiddleware(
       `<${WEBFONT_URL}/ll-unica77_regular.woff2>; rel=preload; as=font; crossorigin`,
       `<${WEBFONT_URL}/ll-unica77_medium.woff2>; rel=preload; as=font; crossorigin`,
       `<${WEBFONT_URL}/ll-unica77_italic.woff2>; rel=preload; as=font; crossorigin`,
-      ...linkHeaders,
```

Comment:
Things have been crazy today and i haven't been following closely, but are you sure that removing `defer` in collectAssets doesn't address all the things in the best way? 

We're loading _so much_ js. We want to hint that.

Reply:
To be honest, no I'm not sure. My tests were pretty clear that the js should stay deferred ("lab score" went from 70 to 55 otherwise). This change eliminated the warnings, but didn't score any better.

If it were the middle of the week, I'd say let's try it for an official Calibre run or 2. Since it's not, let's wait and revisit next week. I think the more impactful change will be to preload the main image (https://github.com/artsy/force/pull/14974), ahead of these assets.

Reply:
We can run a couple manual runs, but i see what you mean in terms of getting the weeks average. Either way!